crates/forge/bin/cmd/test/mod.rs

```diff
@@ -692,6 +693,7 @@ async fn test(
             let mut builder = CallTraceDecoderBuilder::new()
                 .with_labels(result.labeled_addresses.iter().map(|(a, s)| (*a, s.clone())))
                 .with_events(local_identifier.events().cloned())
+                .with_functions(local_identifier.functions().cloned())
```

Comment:
Can also add a helper function for local identifier that calls these 2 other methods